doc/users/next_whats_new/color_support_for_math_to_image.rst

```diff
@@ -0,0 +1,11 @@
+``math_to_image`` now has a *color* keyword argument
+--------------------------------------------------------
+
+To easily support external libraries that rely on the rendering of Matplotlib
+to generate figures, a *color* keyword argument was added to
+``math_to_image``.
```

Comment:
Oh, this should be a cross-reference as well (but no the one in the title.)

Reply:
Makes sense! I also added a more verbose explanation.